buidl/script.py

```diff
@@ -613,11 +613,11 @@ def address_to_script_pubkey(s):
         # p2sh
         h160 = decode_base58(s)
         return P2SHScriptPubKey(h160)
-    elif s[:4] in ("bc1q", "tb1q"):
+    elif s[:4] in ("bc1q", "tb1q") or s[:6] in ("bcrt1q",):
         if len(s) == 42:
             # p2wpkh
             return P2WPKHScriptPubKey(decode_bech32(s)[2])
-        elif len(s) == 62:
+        elif len(s) == 62 or len(s) == 64:
```

Comment:
I think this is fuzzy. I think a root-level branch specific for regtest would be better because a `bc1q` with length 64 should throw.

```
...
elif s[:4] in ("bc1q", "tb1q"):
        if len(s) == 42:
            # p2wpkh
            return P2WPKHScriptPubKey(decode_bech32(s)[2])
        elif len(s) == 62:
            # p2wskh
            return P2WSHScriptPubKey(decode_bech32(s)[2])
elif s[:6] == "bcrt1q" and len(s) == 64:
    # p2wsh
    return P2WSHScriptPubKey(decode_bech32(s)[2])
...
```